src/main/resources/assets/minecolonies/lang/en_us.json

```
@@ -343,28 +343,32 @@
   "item.minecolonies.scroll.colony.tip": "Colony: %s",
   "item.minecolonies.scroll.colony.none": "None, set one with sneak + right-click",
   "item.minecolonies.scroll_area_tp": "Ultrasafe colony group-teleport scroll",
-  "item.minecolonies.scroll_area_tp.tip": "Teleport yourself and nearby players to the colony",
+  "item.minecolonies.scroll_area_tp.tip": "Teleports yourself and nearby players to the colony",
```

It was describing what the scroll does, not what you do. It is implying the subject:
"[the scroll] teleports yourself ..."
But you're right that it sounds a little odd
````suggestion
  "item.minecolonies.scroll_area_tp.tip": "Teleports you and nearby players to the colony",
````